modules/nf-core/truvari/consistency/main.nf

```diff
@@ -0,0 +1,51 @@
+
+process TRUVARI_CONSISTENCY {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/truvari:4.1.0--pyhdfd78af_0':
+        'biocontainers/truvari:4.1.0--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(vcf1), path(vcf2), path(vcf3)
+
+    output:
+    tuple val(meta), path("*.consistency") , emit: consistency
+    path "versions.yml"                    , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def vcf3   = vcf3 ?: "$vcf3"
+
+    """
+    truvari \\
+        consistency \\
+        $args \\
+        $vcf1 \\
+        $vcf2  \\
+        $vcf3 > ${prefix}.consistency
```

Comment:
How is the output extension defined here? I would keep it at `.txt` or `.json` instead of a new unknown extension :)

Reply:
it is either json or txt depending on the arguments given, that is why I couldn't decide.

Reply:
You can parse the `args` to see which one it is. that way you can give the right extension to the output. See here for an example of how we do this in bcftools: https://github.com/nf-core/modules/blob/5074bd37c59454497e790d9210e2f7a876f2c24f/modules/nf-core/bcftools/sort/main.nf#L23-L27

Reply:
done :)